src/main/kotlin/com/wgslfuzz/semanticspreservingtransformations/GenerateArbitraryStatements.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2025 The wgsl-fuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wgslfuzz.semanticspreservingtransformations
+
+import com.wgslfuzz.core.Scope
+import com.wgslfuzz.core.ShaderJob
+import com.wgslfuzz.core.Statement
+
```

Comment:
A general thought on generating arbitrary statements - should there be an augmented-AST way to indicate "these statements are arbitrary; reducer, you can do what you want with them?" With the current design of wrapping would the reducer be able to easily figure out that such statements could be removed?

This doesn't have to be sorted now but a note could be added in #186.